### DISCUSSION TITLE
Idea and Proposal: Display "PV Display Name" when using the Cross-Hair Cursor in the Data Browser

### DISCUSSION THREAD

**Discussion by abrahamwolk** [ControlSystemStudio/phoebus#3095] [Ideas]
It could be helpful if the Display Name of each PV is displayed when using the cross-hair cursor in the Data Browser, especially when there are a large number of PVs in the Data Browser:

![screenshot](https://github.com/user-attachments/assets/a82c0785-2e85-489c-9295-d68ea9b16782)

The idea and proposal is to add the Display Name in this way to the cross-hair cursor in the Data Browser.

**Comment by kasemir** [comment]
Maybe just use the display name, not both PV and display name.

The display name defaults to the PV name, and in our use here it typically stays that way.
.. which is fine, so by default you see the PV name, looks as before.

But if you decide to enter something else for the display name, then it's what you see in the cursor marker.

**Comment by abrahamwolk** [comment]
> Maybe just use the display name, not both PV and display name.

Yes, I agree. The idea is to just use the display name, not the PV name.

**Comment by abrahamwolk** [comment]
I have created a PR for this feature here: https://github.com/ControlSystemStudio/phoebus/pull/3096
